mapbox/libandroid-telemetry/src/main/java/com/mapbox/services/android/telemetry/constants/TelemetryConstants.java

```diff
@@ -10,7 +10,8 @@
 
 public class TelemetryConstants {
 
-  public static final String OPERATING_SYSTEM = "Android - " + Build.VERSION.RELEASE;
+  public static final String PLATFORM = "Android";
+  public static final String OPERATING_SYSTEM = PLATFORM + " - " + Build.VERSION.RELEASE;
```

Comment:
The current iOS Maps SDK omits the `-` (`iOS 10.3.1`); @mick specified the `-` so @boundsj is that an iOS telem bug?